spec.bs

```diff
@@ -227,6 +227,32 @@ spec: html; urlPrefix: https://www.rfc-editor.org/rfc/
         1. [=set/Append=] |callerOrigin| to |topicsHistoryEntry|'s [=topics caller origins=].
   </div>
 
+  <h2 id="derive-top-5-topics-header">Derive top 5 topics</h2>
+
+  Given a [=list=] of [=topics history entries=] <var ignore=''>historyEntriesForUserTopics</var>, the browser should provide an algorithm to <dfn>derive top 5 topics</dfn>, that are believed to be valuable for the Topics callers.
+
+  <div class="note">
+  Chrome's initial release scores topics by the frequency of page loads with that topic.
+
+    <div class="example">
+      <div algorithm="example-algorithm">
+
+      Given a [=list=] of [=topics history entries=] |historyEntriesForUserTopics|:
+        1. Let |topicsCount| be an empty map.
+        1. For each [=topics history entry=] |historyEntry| in |historyEntriesForUserTopics|:
+            1. [=Classify=] |historyEntry|'s [=topics history entry/topics calculation input data=] into |topicIds|.
+            1. For each |topicId| in |topicIds|:
+              1. If |topicsCount|[|topicId|] does not exist:
```

Comment:
In the output at https://pr-preview.s3.amazonaws.com/xyaoinum/topics/pull/203.html#derive-top-5-topics this indentation got lost. I think it's just missing a space.